trailblazer/services/slurm/slurm_service.py

```diff
@@ -0,0 +1,9 @@
+from abc import ABC, abstractmethod
+
+from trailblazer.constants import TrailblazerStatus
+
+
+class SlurmService(ABC):
```

Comment:
Might be better to call this one `WorkflowManagerService` and create a service for tower as well implementing it.